src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -115,6 +116,21 @@ class Config:
         " tokens is generated). Set to `None` to ignore this parameter."
         " Default is `None`.",
     )
+    top_p: Optional[float] = Field(
```

Comment:
Let's be more verbose about those arguments. Put yourself in the shoes of a user, they need to have a good documentation of what those arguments do.

Reply:
name is fine, its more about the description